config/config.go

```diff
@@ -1435,6 +1435,12 @@ var (
 	// and label name characters that are not alphanumerics/underscores to underscores.
 	// Unit and type suffixes may be appended to metric names, according to certain rules.
 	UnderscoreEscapingWithSuffixes translationStrategyOption = "UnderscoreEscapingWithSuffixes"
+	// (EXPERIMENTAL): NoTranslation disables all translation of incoming metric
```

Comment:
```suggestion
	// NoTranslation (EXPERIMENTAL): disables all translation of incoming metric
```

Maybe that's clear enough and also makes the linter happy?

Reply:
applied manually